Automattic/wp-calypso
# User management: Add Viewers tab

Closes https://github.com/Automattic/wp-calypso/issues/77814.

## Proposed Changes

This PR separates viewers from the other types of users on a site. It adds a new tab called "Viewers", which is (will be, TODO item) searchable through e-mail or username.

![image](https://github.com/Automattic/wp-calypso/assets/26530524/38bf60a5-a6e1-4c39-b438-45ac4417f93b)

## TO DO

- [x] Input placeholder
- [x] CTA color and text
- [x] Pre-select the "Viewer" role when clicking the CTA that sends the user to `/people/new/%s?role=viewer`
- [x] Check if the infinite query works
- [x] Remove the `include_viewers` query parameter from the `/sites/%s/users` endpoint (do we want to do that?)
- [ ] Update [support docs](https://wordpress.com/support/invite-people/#adding-and-removing-viewers-on-a-private-site) and heads-up HEs

## Testing Instructions

In a private site, navigate to `Users > All Users`.

Verify that the "Viewers" tab shows up.

Verify that a new user shows up in the list after adding a new user with the viewer role.
